src/app/public/modules/radio/radio.component.spec.ts

```diff
@@ -66,12 +66,24 @@ describe('Radio component', function () {
   });
 
   describe('Standard radio component', () => {
-    beforeEach(fakeAsync(function () {
+    let testComponent: SkyRadioTestComponent;
+
+    beforeEach(fakeAsync(() => {
       fixture = TestBed.createComponent(SkyRadioTestComponent);
 
+      componentInstance = fixture.componentInstance;
+      testComponent = fixture.debugElement.componentInstance;
+
       fixture.detectChanges();
       tick();
-      componentInstance = fixture.componentInstance;
+    }));
+
+    it('should emit the new disabled value when it is modified', fakeAsync(() => {
```

Comment:
I don't see any `tick()`s or fake async functions in this test... do we really need to use `fakeAsync` here?

Reply:
Removed it, test passes. Guess not! Fixed in last commit